### DISCUSSION TITLE
Eerste slash uit 'FullName' weglaten

### DISCUSSION THREAD

**Discussion by jipclaassens** [ObjectVision/GeoDMS#20] [Ideas]
Eerste slash uit 'FullName' weglaten

trello issue #158

**Comment by eoudejans** [comment]
Often the first slash in he FullName is used to indicate that the path starts from the root. This convention can for instance be found in the POSIX.1-2008 standard which says:

"A pathname consisting of a single / shall resolve to the root directory of the process. A null pathname shall not be successfully resolved."

**Comment by jipclaassens** [comment]
If you omit the first slash, it could be a relative path. That is ambiguous.

We have decided to not implement this.
